apps/shipments/serializers.py

```diff
@@ -386,3 +388,89 @@ class Meta:
 
     def create(self, validated_data):
         return TrackingData.objects.create(**validated_data, **self.context)
+
+
+class ChangesDiffSerializer:
+    def __init__(self, queryset, request):
+        self.queryset = queryset
+        self.request = request
+
+        self.relation_fields = ('ship_from_location', 'ship_to_location', 'final_destination_location',
```

Comment:
Does it make sense to use RELATED_FIELDS_WITH_HISTORY_MAP.keys() for this?

Reply:
The wording will definitely make more sense. But I don't understand what will we be mapping those fields to?